source/images/homepage/open-quote.svg

```diff
@@ -0,0 +1 @@
+<svg width="49" height="41" xmlns="http://www.w3.org/2000/svg"><path d="M49 18.923V41H26.782V27.05c0-5.174.94-9.865 2.822-14.07 1.88-4.205 5.024-8.532 9.428-12.98l7.926 6.55c-4.244 4.287-6.887 8.412-7.926 12.373H49zm-26.782 0V41H0V27.05c0-5.174.94-9.865 2.822-14.07C4.702 8.774 7.846 4.447 12.25 0l7.926 6.55c-4.245 4.287-6.887 8.412-7.926 12.373h9.968z" fill="#FF4F5E" fill-rule="evenodd"/></svg>
```

Comment:
I'm curious about this one, and a better frontend dev than me would have an answer for this — are the `height` and `width` attributes useful here or is this better left uncontrolled and letting CSS do the work?

Reply:
Good point! I think it depends on how the graphic was designed and which application was used to generate the SVG, so each case is different (sometimes having the `viewBox` attribute is essential). I haven't been paying much attention to SVG's source code. Usually, I just get the SVG file from designers, optimize it with `svgo`, and commit the file. 

For this case here, going with your suggestion will definitely work and save us extra bits of code. However, from a process perspective, I would like to leave SVG files itself to designers and `svgo` since they are better experts. Plus, I think if I keep doing these manual code optimization work, one day I will accidentally mess up the graphic somehow without noticing it 😂

Reply:
That said, please file a ticket if you have any suggestions for improving this process. It's been me doing most the front end work so my opinions can be completely biased. 😁